src/platform/datapath_kqueue.c

```diff
@@ -865,7 +888,11 @@ CxPlatSocketContextInitialize(
         setsockopt(
             SocketContext->SocketFd,
             ForceIpv4 ? IPPROTO_IP : IPPROTO_IPV6,
+#if defined(IP_RECVPKTINFO)
             ForceIpv4 ? IP_RECVPKTINFO : IPV6_RECVPKTINFO,
+#elif defined(DSTADDR_SOCKOPT)
```

Comment:
```suggestion
#else
```
It must be defined, or you throw an error above.